### PR TITLE
Update OIDC Okta instructions

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
Updates the OIDC Okta instructions to add a fourth step under https://docs.sensu.io/sensu-go/latest/operations/control-access/oidc-auth/#oidc-provider-configuration re: configuring authorization via roles/cluster roles and role bindings/cluster role bindings for OIDC users and groups.

## Motivation and Context
Caleb pointed me to https://discourse.sensu.io/t/issues-implementing-oidc-authentication-with-okta/2499, which indicates we need to add a step re: authorization in the Okta setup instructions and confirm the redirect URI format.

### CODE REVIEW

**Comment by hillaryfraley** [comment]
One more question about this PR:

According to [this Discourse post](https://discourse.sensu.io/t/issues-implementing-oidc-authentication-with-okta/2499/3), the user had to update the Okta configuration with:
 
`Initiate login URI: https://our-api-domain-name/api/enterprise/authentication/v2/oidc/authorize?callback=https://our-dashboard-domain-name/a/oidc/callback?redirect-to=/c/~/n/default/events`

I do not see any information in our instructions regarding what entry to make for `Initiate login URI`. Do we need to add this to the instructions in the docs?

**Comment by hillaryfraley** [comment]
@ccressent since you responded to the user question that inspired this PR (https://discourse.sensu.io/t/issues-implementing-oidc-authentication-with-okta/2499) are you the right person to review and address the question in my previous comment?

File: content/sensu-go/5.21/operations/control-access/oidc-auth.md [hunk 1]
```diff
@@ -429,6 +429,12 @@ For example, if you have an Okta group `groups` and you set the `groups_prefix`
 3. Add the `redirect_uri` configuration attribute in the [OIDC scope][25] and set the value to the Redirect URI configured at step 3 of [Create an Okta application][50]:
    - `"redirect_uri": "API_URL/api/enterprise/authentication/v2/oidc/callback"`
 
+4. Configure [authorization][3] via for your OIDC users and groups by creating [roles (or cluster roles)][4] and [role bindings (or cluster role bindings)][13] that map to the user and group names.
```
Comment (ccressent) [inline content/sensu-go/5.21/operations/control-access/oidc-auth.md]: superflous "via" here?

Comment (ccressent) [reply content/sensu-go/5.21/operations/control-access/oidc-auth.md]: The same comment applies to all the versions.

**Review by ccressent — CHANGES_REQUESTED** [review]

**Comment by ccressent** [comment]
RE: `Initiate Login URL`

I've tried to browse Okta's documentation but I'm not 100% sure unfortunately. I don't think it's necessary in our case, but I haven't tested other than observing that a customer of ours was able to login with OIDC while having this field blank in Okta.

File: content/sensu-go/5.21/operations/control-access/oidc-auth.md (hunk 1 above)
Comment (hillaryfraley) [reply content/sensu-go/5.21/operations/control-access/oidc-auth.md]: Yes, thank you!

**Pull request merged**